students/11th/yejikang/js/login.js

```diff
@@ -0,0 +1,20 @@
+const loginButton = document.querySelector('.loginBtn');
+const loginForm = document.querySelector('.loginForm');
+const userId=document.querySelector('.user-id');
+const userPw = document.querySelector('.user-pw');
+
+function init(){
+    loginButton.disabled = true;
+}
+
+init();
```

Comment:
```suggestion
(function init(){
    loginButton.disabled = true;
})()
```

위와 같이 즉시실행함수 형태로 바꿀 수 있겠네요. 아래 링크 참고해보세요 :)
https://developer.mozilla.org/ko/docs/Glossary/IIFE

Reply:
`()`를 이용해서 형태를 바꿔보았습니다 감사합니다!